students/pkoleyni/Lesson02/fizz_Buzz.py

```diff
@@ -0,0 +1,20 @@
+
+def multiples_of_num(num1,num2):
+    if num1%num2 ==0:
+        return True
+    else:
+        return False
+
+def fizz_buzz():
+    for num in range(1,101):
```

Comment:
Per PEP8 you'll want to add a space between these variables - for num in range(1, 101):

Also, for the ones below.